_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -12,11 +12,13 @@ Some peoples' eyes can't tell the difference between certain colours; that's not
 
 Yet, colour blindness is not rare. Some form of colour blindness affects at least one in twenty people; [Wikipedia has a helpful table](https://en.wikipedia.org/wiki/Color_blindness#Epidemiology), based on [a journal article](https://apfmj.biomedcentral.com/articles/10.1186/s12930-014-0010-3). There are [several different forms](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/). The most common, red-green colourblindness, affects around one in twelve men.
```

Comment:
I recommend referencing the term "Color Vision Deficiency" somewhere as that is more of an accurate term somewhere in the post. https://www.aoa.org/patients-and-public/eye-and-vision-problems/glossary-of-eye-and-vision-conditions/color-deficiency

Reply:
I thought about this too, but was concerned it was too much like jargon. Most lay people know someone who is "colourblind", and most people use the term, so it seemed more reader-friendly to stick to the original.

Reply:
I've also addressed this in #543